src/ui/state/app.ts

```diff
@@ -21,6 +21,7 @@ export type ModalOptionsType = {
   view?: string;
   loom?: string;
   comment?: Reply;
+  instructions?: string;
```

Comment:
Mildly surprised ESLint didn't flag this as not being sorted but maybe that check doesn't run on types :)